src/stores/plan.ts

```diff
@@ -35,11 +35,24 @@ export const maxTimeRange: Writable<TimeRange> = writable({ end: 0, start: 0 });
 
 export const viewTimeRange: Writable<TimeRange> = writable({ end: 0, start: 0 });
 
+/* Subscriptions. TODO ordering */
```

Comment:
are you considering reordering this to help the code be a little more understandable with what depends on others?

Reply:
thanks for catching that, will re-order and comment